v22.2/known-limitations.md

```diff
@@ -10,6 +10,73 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
 
 ## New limitations
 
+### DROP OWNED BY does not support drop functions
+
+`DROP OWNED BY` drops all owned objects as well as any grants on objects not owned by the role.
+
+In its current implementation, this statement does not drop functions. Users must drop their functions manually.
+
+[Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/90476)
+
+{% comment %}
+    initial DROP OWNED BY implementation PR/issue:
+    https://github.com/cockroachdb/cockroach/pull/82936
+    https://github.com/cockroachdb/cockroach/issues/55381
+    doc work in progress (to link to after merge):
+    https://github.com/cockroachdb/docs/pull/15094/files
+    https://cockroachlabs.atlassian.net/browse/DOC-4770 
+    review: rafiss (or devadvocado)
+{% endcomment %}
+
+### DROP OWNED BY not supported where role has synthetic privileges
+
+`DROP OWNED BY` drops all owned objects as well as any grants on objects not owned by the role.
+
+In its current implentation, this operation cannot be performed for roles that have synthetic privileges (entries in `system.privileges`).
+
+Instead, you can use `REVOKE SYSTEM` for the relevant privileges the role has in `system.privileges`.
+
+[Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/88149)
+
+{% comment %}
+    initial DROP OWNED BY implementation PR/issue:
+    https://github.com/cockroachdb/cockroach/pull/82936
+    https://github.com/cockroachdb/cockroach/issues/55381
+    doc work in progress (to link to after merge):
+    https://github.com/cockroachdb/docs/pull/15094/files
+    https://cockroachlabs.atlassian.net/browse/DOC-4770 
+    review: rafiss
+{% endcomment %}
+
+### Limitations for user-defined functions (UDFs)
+
+#### Support for user-defined functions
+
+User-defined functions are not supported in:
```

Comment:
Here's some more limitations with UDFs:

1. Statements within a UDF body cannot contain subqueries. https://github.com/cockroachdb/cockroach/issues/87291
2. Mutation statements like `INSERT`, `UPDATE`, `DELETE`, and `UPSERT` are not allowed in UDF bodies. https://github.com/cockroachdb/cockroach/issues/87289
3. CTEs are not allowed in UDF bodies. https://github.com/cockroachdb/cockroach/issues/92961

There's more limitations with UDFs, but they are less _surprising_ and I would consider them additional features more than known limitations.

Reply:
We should also update the list of limitations on the UDF page.